### PR TITLE
Video resources

### PR DESCRIPTION
**Pull request by nexussays**
https://app.asana.com/0/1186780595903796/1198176573006073

https://projects.invisionapp.com/d/main?origin=v7#/console/20428599/434276276/preview?scrollOffset=2985

### CODE REVIEW

**Review by mojowen — DISMISSED** [review]

**Review by mojowen — DISMISSED** [review]

File: site/src/components/page-faq/page-faq.tsx [hunk 1]
```diff
@@ -12,55 +12,53 @@ export class PageFaq {
...
    public render() {
       const data = this.data || [];
-      const isModalOpen = this.isModalOpen;
+      const { isModalOpen } = this;
 
       return (
-      <Host>
-         <h1>{ this.page_title || "" }</h1>
+         <Host>
+            <h1>{this.pageTitle || ""}</h1>
```
Comment (Samcbehrens) [inline site/src/components/page-faq/page-faq.tsx:30]: Is my linting messed up? It's weird that it wouldn't do this with the --fix flag.

**Comment by nexussays** [comment]
Good to merge if approved in https://app.asana.com/0/1186780595903796/1198176573006073

File: site/src/components/page-faq/page-faq.tsx (hunk 1 above)
Comment (nexussays) [reply site/src/components/page-faq/page-faq.tsx:30]: Yeah it seems there are some things it doesn't fix automatically? Variable renaming seems to be one of them. Maybe that's documented somewhere, dunno

File: site/src/components/page-faq/question-section.scss [hunk 2]
```diff
@@ -1,49 +1,50 @@
...
+         transition: transform 0.25s;
+         fill: $color-accent;
+         margin-right: 0.5em;
+         background-color: Transparent;
+         outline: none;
+         border: none;
+         cursor: pointer;
+
+         &.open {
+            transform: rotate(-180deg) !important;
```
Comment (Samcbehrens) [inline site/src/components/page-faq/question-section.scss:23]: why do you want to rotate -180 deg?

**Review by Samcbehrens — DISMISSED** [review]

File: site/src/components/page-faq/question-section.scss (hunk 2 above)
Comment (nexussays) [reply site/src/components/page-faq/question-section.scss:23]: It feels right

**Review by Samcbehrens — DISMISSED** [review]

**Review by Samcbehrens — APPROVED** [review]

**Pull request merged**